src/cms/templates/events/_event_filter_form.html

```diff
@@ -0,0 +1,94 @@
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load content_filters %}
+{% load poi_filters %}
+<form method="post">
+    {% csrf_token %}
+    {% get_current_language as LANGUAGE_CODE %}
+    {% unify_language_code LANGUAGE_CODE as LANGUAGE_CODE %}
+    {% get_language LANGUAGE_CODE as current_language %}
+    <div class="flex flex-wrap p-4 items-start">
+        <div class="w-1/3 pr-6">
+            <h3 class="font-bold mb-3">{% trans 'Date and time' %}</h3>
+            <div class="flex flex-wrap mb-2 items-end">
+                <div class="w-1/2 pr-1">
+                    <label for="after_date" class="mb-2 block font-bold text-sm">{% trans 'Not before' %}</label>
+                    {% render_field filter_form.after_date|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+                <div class="w-1/2 pl-1">
+                    {% render_field filter_form.after_time|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+            </div>
+            <div class="flex flex-wrap items-end">
+                <div class="w-1/2 pr-1">
+                    <label for="before_date" class="mb-2 block font-bold text-sm">{% trans 'Not after' %}</label>
+                    {% render_field filter_form.before_date|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+                <div class="w-1/2 pl-1">
+                    {% render_field filter_form.before_time|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+            </div>
+        </div>
+        <div class="w-1/3 pr-6">
+            <h3 class="font-bold mb-3">{% trans 'Location' %}</h3>
```

Comment:
```suggestion
            <h3 class="font-bold mb-3">{% trans 'Location' %}</h3>
            <label for="poi-query-input" class="mb-2 block font-bold text-sm">{% trans 'Event location' %}</label>
```
I'd suggest to add an extra label here, because
1. It makes it clearer that only event locations can be searched here (and not cities etc.)
2. The input field lines up with the time & date fields

Reply:
Thanks for the hints, I didn't even think about it. This definitely makes the form look better!